persistence/rocksdb/src/main/java/org/infinispan/persistence/rocksdb/metrics/StatisticsExporterImpl.java

```diff
@@ -0,0 +1,701 @@
+package org.infinispan.persistence.rocksdb.metrics;
+
+import org.infinispan.jmx.annotations.MBean;
+import org.infinispan.jmx.annotations.ManagedAttribute;
+import org.rocksdb.Statistics;
+import org.rocksdb.TickerType;
+
+@MBean
```

Comment:
@danberindei Do ```@MBean```s automatically become components? Or do we also need to add ```@Scope``` to force that ? 

Btw @burmanm , jmx export will be a side-effect of this, which is cool, even if you did not intend to export it via jmx.

Reply:
`@Scope` is only needed to allow the component annotations to work, e.g. `@Inject` or `@Start`.

I think I made it this way so that the annotations could be split: the component injection/lifecycle stuff on the implementation, and the JMX/metrics stuff on an interface. But the end result is that the component accessor is generated for any class/interface with an `@MBean` annotation, and the component registry doesn't require any lifecycle/injection either.